changes/5970.removed

```diff
@@ -0,0 +1 @@
+Removed indentations for PrefixTable in the child prefixes tab of PrefixDetailView, GlobalSearchView and VLANDetailView
```

Comment:
I'd vote for `.changed` here, it's not removed functionality, just a change in behavior.